controllers/enrollmentkeys.go

```diff
@@ -10,11 +10,11 @@ import (
 	"github.com/google/uuid"
 	"github.com/gorilla/mux"
 
-	"github.com/gravitl/netmaker/auth"
 	"github.com/gravitl/netmaker/logger"
 	"github.com/gravitl/netmaker/logic"
 	"github.com/gravitl/netmaker/models"
 	"github.com/gravitl/netmaker/mq"
+	"github.com/gravitl/netmaker/pro/auth"
```

Comment:
there shouldn't be any reference to pro from community

Reply:
@abhishek9686 , I moved some of the common functions back to ce.  Please have a review again. Thanks.